### PR TITLE
chore(NX-3492): change support links from ZD to Salesforce

### PR DESCRIPTION
**Pull request by araujobarret**
The type of this PR is: **Chore**

This PR solves [NX-3492]

### Description

Changes Support links from Zendesk to Salesforce.

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[NX-3492]: https://artsyproduct.atlassian.net/browse/NX-3492?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lidimayra — APPROVED** [review]

**Comment by lidimayra** [comment]
2 of the links added in this PR are not working as expected. These URLs are matching the ones we have in the [spreadsheet](https://docs.google.com/spreadsheets/d/1Yofy_cExeEgdV1-o8rzMrGnJ6In4c0aotV5dzowhYUU/edit#gid=571260299). I'll share this on [#zendesk-sf-migration](https://artsy.slack.com/archives/C04U9SJ5A0K) channel

- https://support.artsy.net/s/article/What-Counts-as-an-Artworks-Proof-of-Authenticity
- https://support.artsy.net/s/article/How-are-taxes-customs-VAT-and-import-fees-handled-on-works-listed-with-secure-checkout
- https://support.artsy.net/s/article/How-do-I-place-a-bid-in-an-auction
- https://support.artsy.net/s/topic/0TO3b000000UesxGAC/sell
- https://support.artsy.net/s/article/Im-an-artist-Can-I-submit-my-own-work-to-sell
- https://support.artsy.net/s/topic/s/topic/0TO3b000000UevEGAS/contacting-a-gallery ❌
- https://support.artsy.net/s/article/How-to-Complete-Identity-Verification
- https://support.artsy.net/s/topic/s/topic/0TO3b000000UevOGAS/sell-with-artsy ❌
- https://support.artsy.net/s/article/The-Artsy-Guarantee
- https://support.artsy.net/s/topic/0TO3b000000UessGAC/buy
- https://support.artsy.net/s/article/The-Complete-Guide-to-Auctions-on-Artsy

File: src/Apps/Conversation/Components/DetailsSidebar.tsx [hunk 1]
```diff
@@ -199,7 +199,7 @@ export const DetailsSidebar: FC<DetailsProps> = ({
             Support
           </Text>
           <RouterLink
-            to={`https://support.artsy.net/hc/en-us/sections/360008203054-Contact-a-gallery`}
+            to="https://support.artsy.net/s/topic/s/topic/0TO3b000000UevEGAS/contacting-a-gallery"
```
Comment (lidimayra) [inline src/Apps/Conversation/Components/DetailsSidebar.tsx:202]:
```suggestion
            to="https://support.artsy.net/s/topic/0TO3b000000UevEGAS/contacting-a-gallery"
```

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWAHowItWorksModal.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ export const MyCollectionArtworkSWAHowItWorksModal: React.FC<{
   onClose: () => void
 }> = ({ onClose }) => {
   const article =
-    "https://support.artsy.net/hc/en-us/sections/360008311913-Sell-with-Artsy"
+    "https://support.artsy.net/s/topic/s/topic/0TO3b000000UevOGAS/sell-with-artsy"
```
Comment (lidimayra) [inline src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWAHowItWorksModal.tsx:8]:
```suggestion
    "https://support.artsy.net/s/topic/0TO3b000000UevOGAS/sell-with-artsy"
```

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx [hunk 3]
```diff
@@ -42,7 +42,7 @@ export const MyCollectionArtworkSWASectionSubmitted: React.FC<Props> = ({
   }
 
   const article =
-    "https://support.artsy.net/hc/en-us/sections/360008311913-Sell-with-Artsy"
+    "https://support.artsy.net/s/topic/s/topic/0TO3b000000UevOGAS/sell-with-artsy"
```
Comment (lidimayra) [inline src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx:45]:
```suggestion
    "https://support.artsy.net/s/topic/0TO3b000000UevOGAS/sell-with-artsy"
```

**Review by lidimayra — CHANGES_REQUESTED** [review]

**Review by lidimayra — APPROVED** [review]

**Comment by araujobarret** [comment]
Thanks @lidimayra, changed the broken links in https://github.com/artsy/force/pull/12274/commits/cb0c350f9ad46aed6071cfc3e6d80d2b441823d1

**Review by MrSltun — APPROVED** [review]
✨ 🚀

**Pull request merged**